inginious/common/tasks_problems.py

```diff
@@ -181,6 +181,18 @@ def input_is_consistent(self, task_input, default_allowed_extension, default_max
 
     @classmethod
     def parse_problem(self, problem_content):
+        # Checking problem edit inputs
+        if len(problem_content["offset"]) == 0:
+            problem_content["offset"] = 1 
```

Comment:
```suggestion
            del problem_content["offset"]
```

This ease YAML file reading and saves some space.